### DISCUSSION TITLE
The Korean language model fails to recognize '.' because it is missing.

### DISCUSSION THREAD

**Discussion by junhwan-lim** [PaddlePaddle/PaddleOCR#13144] [Q&A] [answered]
The Korean language model fails to recognize '.' because it is missing.

**Comment by UserWangZz** [comment, marked as answer]
Can you provide detailed questions？

**Reply by junhwan-lim** [reply]
When using the PaddleOCR Korean model, the korean_dict.txt file does not include the "." character, and the model fails to recognize "." during actual use. This causes difficulties in reading date information such as expiration dates.

**Reply by uyeongjae** [reply]
Comma(`,`) also not included in the korean dictionary(`korean_dict.txt`).
Is there a reason to be excluded?
